python/tvm/relay/op/_transform.py

```diff
@@ -189,10 +192,9 @@ def _reshape_shape_func(data_shape, newshape, ndim):
             out[infer_idx] = old_size // new_size
     return out
 
-@_reg.register_shape_func("reshape", False)
+@_reg.register_shape_func("reshape", True)
```

Comment:
I still have a bit concern here because if `newshape` is not a tensor input, we can still treat `reshape` as data independent shape func, which means that we can fuse reshape with other ops. But now, if the output of `reshape` is dynamic, it can no longer be fused any more.

Reply:
In the short term, is it possible that we just check reshape in fuse to identify constant newshape case? In the future, maybe we would like to have a more systematic to way set fuse pattern regarding the dynamism of op inputs.

Reply:
@icemelon9  I've solved your concern. Could you please take another look? Thanks